dataactvalidator/interfaces/validatorValidationInterface.py

```diff
@@ -272,13 +272,16 @@ def addMultiFieldRule(self,fileId, ruleTypeText, ruleTextOne, ruleTextTwo, descr
             ruleTextOne: definition of rule
             ruleTextTwo: definition of rule
             description: readable explanation of rule
+            ruleLabel: description of rule
```

Comment:
Thanks for adding these.  Minor point, the rule label doesn't provide any description, it's a string like "A23" that provides a label we can use to refer to that rule.